Automattic/jetpack
# Tiled Galleries: fix dynamic property deprecation notices

Follow-up from 30786

## Proposed changes:

Fix more deprecation notices introduced with PHP 8.2:

```
PHP Deprecated:  Creation of dynamic property Jetpack_Tiled_Gallery_Rectangular_Item::$img_src is deprecated in modules/tiled-gallery/tiled-gallery/tiled-gallery-item.php on line 56
PHP Deprecated:  Creation of dynamic property Jetpack_Tiled_Gallery_Rectangular_Item::$img_src_grayscale is deprecated in modules/tiled-gallery/tiled-gallery/tiled-gallery-item.php on line 134
PHP Deprecated:  Creation of dynamic property Jetpack_Tiled_Gallery_Rectangular_Item::$size is deprecated in modules/tiled-gallery/tiled-gallery/tiled-gallery-item.php on line 136
```

### Other information:


## Jetpack product discussion

Primary issue: #27927

## Does this pull request change what data or activity we track or use?

* No

## Testing instructions:

On a site connected to WordPress.com:

* Go to Posts > Add New and add a Tiled Gallery block.
* Ensure such galleries are displayed properly on the frontend.
